app/src/main/res/layout/swipeactions_picker.xml

```diff
@@ -1,10 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.gridlayout.widget.GridLayout
+<ScrollView
     xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:grid="http://schemas.android.com/apk/res-auto"
-    android:id="@+id/pickerGridLayout"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:padding="16dp"
-    grid:columnCount="2"
-    grid:alignmentMode="alignBounds" />
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="wrap_content"
```

Comment:
Shouldn't this be match_parent to ensure that the layout takes the full width?

Reply:
The view inside this layout has its width set to match_parent, which will automatically cause the parent layout's width to also be match_parent. 
So i Suppose this not to be an issue.